kickstarts/base.ks.erb

```diff
@@ -3,6 +3,7 @@
 logging --level=debug
 
 #version=DEVEL
+text
```

Comment:
It's for debugging and slightly faster builds, which is useful when you're doing a lot of builds :) It does not impact the image size. Happy to separate it out in a separate patch, lmk.